src/morphological_tagger/MorphWriter.cpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2012-2015 Andre Martins
```

Comment:
You need to implement this function. You are currently not writing the predicting CONLL files?